build-system/compile/bundles.config.js

```diff
@@ -1170,6 +1170,13 @@ exports.extensionBundles = [
     latestVersion: '0.1',
     type: TYPES.MEDIA,
   },
+  {
+    name: 'amp-moviads-player',
+    version: '0.1',
+    latestVersion: '0.1',
+    options: {hasCss: true},
+    type: TYPES.MISC,
+  },
```

Comment:
This file doesn't contain this configuration anymore, please update your branch and move to `bundles.config.extensions.json` in JSON, without `type`.

```json
  {
    "name": "amp-moviads-player",
    "version": "0.1",
    "latestVersion": "0.1",
    "options": {"hasCss": true}
  },
```

Reply:
changed type